boilerplate/openshift/golang-osd-operator-osde2e/e2e-harness-generate.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+set -e
+
+cmd=${0##*/}
+
+usage() {
+    cat <<EOF
+Usage: $0 OPERATOR_NAME OSDE2E_CONVENTION_DIR
+ 
+EOF
+    exit -1
+}
+
+OPERATOR_NAME=$1
+OSDE2E_CONVENTION_DIR=$2
+REPO_ROOT=$(git rev-parse --show-toplevel)
+HARNESS_DIR=$REPO_ROOT/osde2e
```

Comment:
Bit nit picky, is there a reason we are using a top level `osde2e` directory and not placing this inside of a `test`/`tests` directory?

`test/osde2e` or whatever would be my preference

https://github.com/kubernetes/kubernetes/tree/master/test
https://github.com/openshift/origin/tree/master/test